types/src/front/lib/core_api.ts

```diff
@@ -1208,12 +1214,49 @@ export class CoreAPI {
     return this._resultFromResponse(response);
   }
 
+  private async _fetchWithError(
+    url: string,
+    init?: RequestInit
+  ): Promise<Result<{ response: Response; duration: number }, CoreAPIError>> {
+    const now = Date.now();
+    try {
+      const res = await fetch(url, init);
+      return new Ok({ response: res, duration: Date.now() - now });
+    } catch (e) {
+      const duration = Date.now() - now;
+      const err: CoreAPIError = {
+        code: "unexpected_network_error",
```

Comment:
This makes the error harder to read. Why not just set the error message to `CoreAPI error - network error` ?

Reply:
Updated it